src/app.js

```diff
@@ -1,88 +1,167 @@
+
+function onClick(element) {
+  document.getElementById('img01').src = element.src;
+  document.getElementById('modal01').style.display = 'block';
+  const captionText = document.getElementById('caption');
+  captionText.innerHTML = 'PRICE: $10';
+}
+
+li1.onclick = function () {
+  document.getElementById('img_div').outerHTML = '';
+  createDiv();
+  const menu = document.createElement('div');
+  const imgDiv = document.getElementById('img_div');
+  menu.classList.add(...['d-flex', 'flex-wrap', 'justify-content-center', 'w-50', 'mx-auto', 'pt-5']);
+  imgDiv.appendChild(menu);
+  const card1 = document.createElement('img');
+  card1.id = card1;
+  card1.classList.add('m-5');
+  card1.src = 'https://www.chimuadventures.com/blog/wp-content/uploads/2016/06/Ceviche_Peru_shutterstock1.jpg';
+  card1.style.height = '300px';
+  card1.style.width = '300px';
+  card1.style.objectFit = 'cover';
+  card1.appendChild(document.createTextNode('The best food in the world'));
+  menu.appendChild(card1);
+  card1.onclick = function () { onClick(this); };
+  const card2 = document.createElement('img');
+  card2.classList.add('m-5');
+  card2.src = 'https://www.chimuadventures.com/blog/wp-content/uploads/2016/06/Ceviche_Peru_shutterstock1.jpg';
+  card2.style.height = '300px';
+  card2.style.width = '300px';
+  card2.style.objectFit = 'cover';
+  menu.appendChild(card2);
+  card2.onclick = function () { onClick(this); };
+  const card3 = document.createElement('img');
+  card3.classList.add('m-5');
+  card3.src = 'https://www.chimuadventures.com/blog/wp-content/uploads/2016/06/Ceviche_Peru_shutterstock1.jpg';
+  card3.style.height = '300px';
+  card3.style.width = '300px';
+  card3.style.objectFit = 'cover';
+  menu.appendChild(card3);
+  card3.onclick = function () { onClick(this); };
+  const card4 = document.createElement('img');
+  card4.classList.add('m-5');
+  card4.src = 'https://www.chimuadventures.com/blog/wp-content/uploads/2016/06/Ceviche_Peru_shutterstock1.jpg';
+  card4.style.height = '300px';
+  card4.style.width = '300px';
+  card4.style.objectFit = 'cover';
+  menu.appendChild(card4);
+  card4.onclick = function () { onClick(this); };
+  const card5 = document.createElement('img');
+  card5.classList.add('m-5');
+  card5.src = 'https://www.chimuadventures.com/blog/wp-content/uploads/2016/06/Ceviche_Peru_shutterstock1.jpg';
+  card5.style.height = '300px';
+  card5.style.width = '300px';
+  card5.style.objectFit = 'cover';
+  menu.appendChild(card5);
+  card5.onclick = function () { onClick(this); };
+  const card6 = document.createElement('img');
+  card6.classList.add('m-5');
+  card6.src = 'https://www.chimuadventures.com/blog/wp-content/uploads/2016/06/Ceviche_Peru_shutterstock1.jpg';
+  card6.style.height = '300px';
+  card6.style.width = '300px';
+  card6.style.objectFit = 'cover';
+  menu.appendChild(card6);
+  card6.onclick = function () { onClick(this); };
+};
 
+li2.onclick = function () {
```

Comment:
- [ ] You should use **arrow functions** here instead of **traditional keywords** to be compliant with `ES6` standards.